fe1-web/src/features/wallet/screens/WalletSetSeed.tsx

```diff
@@ -21,14 +20,18 @@ const styles = StyleSheet.create({
 /**
  * Wallet screen to set an already existing mnemonic
  */
-const WalletSetSeed = ({ navigation }: IPropTypes) => {
+const WalletSetSeed = () => {
   /* used to set the mnemonic seed inserted by the user */
   const [seed, setSeed] = useState('');
+  const navigation = useNavigation<any>();
 
   const initWallet = async () => {
     try {
       await Wallet.importMnemonic(seed);
-      navigation.navigate(STRINGS.navigation_synced_wallet);
+      navigation.reset({
```

Comment:
I'm okay with this, but I find it a bit odd that we need to use `reset` though. 
What's wrong with `navigate` ? I'm not sure I understand.

Reply:
Yes,it is odd, the thing is I still do not understand exactly why the bug was happening, navigate should move the selected screen on top of the stack, and the wallet screen stack should not change even when the "parent" navigator(the one used for changing tabs) changes. But it did for whatever reason. 
I figured that maybe resetting the routes history would help, but that still does not explain the bug.